gym_autoencoder/envs/auto_basic_env.py

```diff
@@ -0,0 +1,64 @@
+import importlib
+
+import cv2
+import gym
+import numpy as np
+import torch
+from gym.spaces import Box
+
+
+class AutoencoderBasicEnv(gym.Env):
+    metadata = {'render.modes': ['human']}
+
+    def __init__(self, path_stub, path_model ,shape):
+        self.env = gym.make("procgen:procgen-heist-v0", use_backgrounds=False, render_mode="rgb_array",
+                            distribution_mode="memory")
+        self.model = importlib.import_module(path_stub).Autoencoder()
+        self.model.load_state_dict(torch.load(path_model))
```

Comment:
Wenn man nur eine CPU hat failt das laden hier:

```
RuntimeError: Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU.
```
Wäre cool wenn man das als Parameter irgendwie regeln könnte.

```suggestion
        self.model.load_state_dict(torch.load(path_model, map_location=torch.device("cpu")))
```